totalRP3/Resources/InterfaceIcons.lua

```diff
@@ -84,6 +84,8 @@ TRP3_InterfaceIcons = {
 	BloodElfMale             = { "achievement_character_bloodelf_male", "inv_weapon_bow_38", DEFAULT_ICON_NAME },
 	DarkIronDwarfFemale      = { "ability_racial_foregedinflames", DEFAULT_ICON_NAME },
 	DarkIronDwarfMale        = { "ability_racial_fireblood", DEFAULT_ICON_NAME },
+	DracthyrFemale           = { "inv_dracthyrhead01", "inv_dracthyrhead11"},
+	DracthyrMale             = { "inv_dracthyrhead02", "inv_dracthyrhead16"},
```

Comment:
Same for this one, but as with most things, Meo or Sol will have to give you the final approval.